specs/agents/tracing-spans-handling-huge-traces.md

```diff
@@ -0,0 +1,43 @@
+[Agent spec home](README.md) > [Handling huge traces](tracing-spans-handling-huge-traces.md)
```

Comment:
It sounds interesting - I think it is worth trying out. I wonder if it would be better to do this move after we finalize the content - I'm not sure if Github will track pending comments correctly if the files are moved.